VEHICLE/AttachVehicleToCargobob.md

```diff
@@ -5,18 +5,47 @@ ns: VEHICLE
 
 ```c
 // 0x4127F1D84E347769 0x607DC9D5
-void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle vehicle, Vehicle cargobob, int p2, float x, float y, float z);
-```
-
-```
-p2 is unknown and is always -1 in the script natives.
+void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle cargobob, Vehicle vehicle, int vehicleBoneIndex, float x, float y, float z);
 ```
 
 ## Parameters
-* **vehicle**: The vehicle which should be attached
 * **cargobob**: The cargobob
-* **p2**:
-* **x**:
-* **y**:
-* **z**:
+* **vehicle**: The vehicle which should be attached
+* **vehicleBoneIndex**: A Vehicle bone the hook/magnet should attach to (Can also be false) https://docs.fivem.net/natives/?_0xFB71170B7E76ACBA
+* **x**: x hook/magnet Offset 
+* **y**: y hook/magnet Offset
+* **z**: z hook/magnet Offset
+
+
+LUA Example:
```

Comment:
For the examples section, you don't need this, since examples are already sub-categorized depending on which code block you use, take [this](https://github.com/citizenfx/natives/blob/9a25e674ddf88af3f1575cf499ad50a6eb61d2d2/AUDIO/IsVehicleRadioEnabled.md?plain=1#L22) as an example, which translates into [this](https://docs.fivem.net/natives/?_0x0BE4BE946463F917).

Reply:
Updated 👍